src/twisted/scripts/twistd.py

```diff
@@ -32,3 +32,6 @@ def run():
 
 
 __all__ = ['run', 'runApp']
+
+
+__name__ == '__main__' and run()
```

Comment:
This should be an `if` statement, though I'm guessing you did this to avoid lowering code coverage?